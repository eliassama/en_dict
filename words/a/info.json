{
  "word": "a",
  "ipa": {
    "us": "ə",
    "uk": "ə"
  },
  "sentences": [
    {
      "en": "A changing world has put pressures on the company.",
      "cn": "日新月异的世界使这家公司感到了压力。"
    },
    {
      "en": "A miracle is something that seems impossible but happens anyway.",
      "cn": "奇迹就是看似不可能，却发生了。"
    },
    {
      "en": "As a group, today's old people are still relatively deprived.",
      "cn": "总的来说，当今的老年人依然相对贫困。"
    },
    {
      "en": "Details are available from the Hon. Sec. A.R. Bushby.",
      "cn": "详情可咨询名誉秘书A.R.布什比。"
    },
    {
      "en": "English has hurt me a thousand times, but I still regard it as my first love.",
      "cn": "英语伤我千百遍，我待英语如初恋。"
    },
    {
      "en": "Harry inherited the house and a sizeable chunk of land.",
      "cn": "哈里继承了这座房子以及相当大的一片土地。"
    },
    {
      "en": "He was confirmed as a member of the Church of England.",
      "cn": "他领受了坚振礼，成为英国国教会的一员。"
    },
    {
      "en": "He was fired from his job after roughing up a colleague.",
      "cn": "他因为对一位同事动粗而被开除了。"
    },
    {
      "en": "He’s a really nice man.",
      "cn": "他真是个好人。"
    },
    {
      "en": "I get paid once a month.",
      "cn": "我的工资一个月发一次。"
    },
    {
      "en": "I have tried to pack a good deal into a few words.",
      "cn": "我尽量言简意赅。"
    },
    {
      "en": "I want to train to be an engineer.",
      "cn": "我想经过培训成为一名工程师。"
    },
    {
      "en": "I was assigned to Troop A of the 10th Cavalry.",
      "cn": "我被派往第10骑兵队的A连。"
    },
    {
      "en": "It needs a good clean.",
      "cn": "这得好好清洗一下。"
    },
    {
      "en": "Liver and kidney are particularly rich in vitamin A.",
      "cn": "肝脏和肾脏富含维生素A。"
    },
    {
      "en": "Remember, keep a positive attitude and good things will happen.",
      "cn": "记住：保持乐观的心态，好事自然会发生。"
    },
    {
      "en": "Suddenly they heard a loud bang.",
      "cn": "他们突然听到砰的一声巨响。"
    },
    {
      "en": "Take a look at this.",
      "cn": "瞧瞧这个。"
    },
    {
      "en": "The Koran recommends fasting as a penance before pilgrimages.",
      "cn": "《古兰经》劝告人们在朝圣前斋戒，以作为一种补赎。"
    },
    {
      "en": "There was a hole in the fence.",
      "cn": "篱笆上有一个洞。"
    },
    {
      "en": "They found a labyrinth of tunnels under the ground.",
      "cn": "他们发现了一处迷宫似的地道。"
    },
    {
      "en": "We have a problem.",
      "cn": "我们遇到了一个麻烦。"
    },
    {
      "en": "Would you like a sandwich?",
      "cn": "来块三明治好吗？"
    },
    {
      "en": "You have to do everything you can. You have to work your hardest. And if you do, if you stay positive, then you have a shot at a silver lining.",
      "cn": "你必须全力以赴，最大限度地去努力。如果你这么做，并且保持乐观，你就会看见乌云背后的幸福线。"
    }
  ],
  "rel_words": [],
  "synos": [],
  "phrases": [],
  "trans": [
    {
      "pos": "abbr",
      "tran_cn": "[物]安（ampere）"
    },
    {
      "pos": "art",
      "tran_cn": "一（个），每一（个），任一（个）",
      "tran_en": "used to show that you are talking about someone or something that has not been mentioned before, or that your listener does not know about"
    },
    {
      "pos": "indefinite article",
      "tran_cn": "一，一个〔用于未曾提及或对方不知道的人或物的名称前〕",
      "tran_en": "used to show that you are talking about someone or something that has not been mentioned before, or that your listener does not know about"
    },
    {
      "pos": "n",
      "tran_cn": "字母A；第一流的；学业成绩达最高标准的评价符号",
      "tran_en": "used to show that you are referring to a general type of person or thing and not a specific person or thing"
    }
  ],
  "exchange": {}
}