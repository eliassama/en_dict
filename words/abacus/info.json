{
  "word": "abacus",
  "ipa": {
    "us": "ˈæbəkəs",
    "uk": "ˈæbəkəs"
  },
  "sentences": [
    {
      "en": "A : So am I. Using an abacus is still a basic accounting.",
      "cn": "我也一样,打算盘仍然是会计的基本技能."
    },
    {
      "en": "But that position did not arise because of Abacus 2007 - AC 1 .",
      "cn": "但这些头寸并非源自Abacus2007 -AC1 交易."
    },
    {
      "en": "Children thinking courses: abacus calculation, English, literature special education, English language.",
      "cn": "儿童思维训练班: 珠算 、 英语 、 文学启智 、 英语口语."
    },
    {
      "en": "Do you know how to work an abacus?",
      "cn": "你会打算盘 吗 ?"
    },
    {
      "en": "Except for the sharp clicking of Chueh - hsin's abacus counters, all was still.",
      "cn": "房里只有算盘珠子的接连的 、 清脆的响声."
    },
    {
      "en": "I could write on paper and use abacus for statistic.",
      "cn": "我可以用纸笔写作，要算统计时就打算盘。"
    },
    {
      "en": "In English we call it an abacus.",
      "cn": "在英语里我们叫它为一个算盘。"
    },
    {
      "en": "In response to declining math skills among their students, some Japanese schools are bringing back an ancient calculating device - the abacus.",
      "cn": "为了改善学生日渐衰退的计算能力，日本的一些学校重新使用起一个古老的计算工具-算盘。"
    },
    {
      "en": "She can reckon rapidly on the abacus.",
      "cn": "她用算盘算得很快."
    },
    {
      "en": "She is a passive person like an abacus, lacking initiative in everything.",
      "cn": "她是个算盘珠似的人, 从来做事都是缺乏主动性的."
    },
    {
      "en": "The abacus fell down on the ground and the beads went everywhere.",
      "cn": "算盘摔到了地上,算盘珠散落一地."
    },
    {
      "en": "The abacus is still in use today Chinatowns abroad.",
      "cn": "现在,已经进入了电子计算机时代,但是古老的算盘仍然发挥着重要的作用."
    },
    {
      "en": "The abacus is the ancestor of the modern computer.",
      "cn": "算盘是现代电脑的原型."
    },
    {
      "en": "The children are learning to do their sums on an abacus.",
      "cn": "孩子们正在学做珠算."
    },
    {
      "en": "The clerk reckoned the expenditure last week with an abacus.",
      "cn": "那位职员用算盘将上周的花销计算出来."
    },
    {
      "en": "The room was very quiet. Even the sound of the abacus was stilled.",
      "cn": "屋子里宁静了片刻,算盘珠子的声音也已经停止了."
    },
    {
      "en": "They do use an abacus when they count, do they?",
      "cn": "他们计算的时候用算盘, 不是 吗 ?"
    },
    {
      "en": "They don't use an abacus when they count, right?",
      "cn": "他们计算时用算盘, 对 吗 ?"
    }
  ],
  "rel_words": [],
  "synos": [],
  "phrases": [],
  "trans": [
    {
      "pos": "n",
      "tran_cn": "算盘，<建>（圆柱）顶板",
      "tran_en": "a frame with small balls that can be slid along on thick wires, used for counting and calculating"
    }
  ],
  "exchange": {
    "pl": [
      "abacuses"
    ]
  }
}