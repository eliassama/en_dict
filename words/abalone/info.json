{
  "word": "abalone",
  "ipa": {
    "us": "ˌæbəˈləʊni",
    "uk": "ˌæbəˈloʊni"
  },
  "sentences": [
    {
      "en": "Cut the Abalone Mushrooms into small pieces. Drain and fry the abalone mushrooms till golden brown.",
      "cn": "将杏鲍菇切成花形后, 然后沥干水份备用,将它炸至金黄色."
    },
    {
      "en": "Ingredient: Chicken, slice abalone green onion , custom made superior sauce, flour.",
      "cn": "成分: 鸡腿肉,香鲍, 青葱, 自制上等调味酱, 面粉."
    },
    {
      "en": "Internal density foam padding and abalone cotton.",
      "cn": "内部填充高密度海绵和九孔棉."
    },
    {
      "en": "Lighting is everything. The highly reflective surface and the abalone shell made this photograph a real challenge to take.",
      "cn": "灯光就是一切。高度反光的表面和鲍鱼壳使这张照片的拍摄成为真正的挑战。"
    },
    {
      "en": "The highly reflective surface and the abalone shell made this photograph a real challenge to take.",
      "cn": "高度反光的表面和鲍鱼壳使这张照片的拍摄成为真正的挑战。"
    },
    {
      "en": "Yeast don't normally do any of those reactions on their own, so Belcher and her students had to engineer them to express genes found in organisms such as the abalone.",
      "cn": "酵母通常不会对二氧化碳等做出自我反应，所以贝尔彻和她的学生们必须改造它们来表达类似鲍鱼机体中的那些基因。"
    }
  ],
  "rel_words": [],
  "synos": [
    {
      "pos": "n",
      "words": [
        {
          "words": [
            "paua",
            "ormer",
            "sea-ear"
          ],
          "tran_cn": "[无脊椎]鲍鱼"
        }
      ]
    }
  ],
  "phrases": [],
  "trans": [
    {
      "pos": "n",
      "tran_cn": "<美>鲍鱼（软体动物）",
      "tran_en": "a kind of  shellfish  which is used as food and whose shell contains  mother-of-pearl "
    }
  ],
  "exchange": {
    "pl": [
      "abalones"
    ]
  }
}