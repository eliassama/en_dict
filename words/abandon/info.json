{
  "word": "abandon",
  "ipa": {
    "us": "əˈbændən",
    "uk": "əˈbændən"
  },
  "sentences": [
    {
      "en": "After careful deliberation, it was agreed to abandon the project.",
      "cn": "经过慎重考虑, 大家同意放弃这个项目."
    },
    {
      "en": "Because of the fog they abandoned their idea of driving.",
      "cn": "因为有雾，他们打消了开车去的念头。"
    },
    {
      "en": "Do not abandon yourself to despair.",
      "cn": "不要悲观失望."
    },
    {
      "en": "Each side declared that it would never abandon its principle.",
      "cn": "各方都说绝不放弃自己的原则."
    },
    {
      "en": "He approached life with reckless abandon–I don't think he himself knew what he was going to do next.",
      "cn": "他以不计后果的放纵态度对待生活–我想他自己都不知道他接下来要做什么。"
    },
    {
      "en": "He has spent money with gay abandon.",
      "cn": "他肆意挥霍钱财。"
    },
    {
      "en": "How could she abandon her own child?",
      "cn": "她怎么能抛弃自己的孩子呢？"
    },
    {
      "en": "I decided not to abandon John when he was in such a desperate position.",
      "cn": "我决定不在约翰走投无路的时候抛弃他。"
    },
    {
      "en": "It would be irresponsible of me not to advise my company to abandon this project.",
      "cn": "若我不建议公司放弃这个项目，那我就是不负责任。"
    },
    {
      "en": "People were shouting and cheering with abandon.",
      "cn": "人们兴高采烈,纵情欢呼."
    },
    {
      "en": "The captain gave the order to abandon ship.",
      "cn": "船长下令弃船。"
    },
    {
      "en": "The game had to be abandoned due to bad weather.",
      "cn": "由于天气不好，比赛不得不中止。"
    },
    {
      "en": "The girls jumped up and down and waved their arms with abandon.",
      "cn": "那些女孩子跳上跳下尽情地挥舞着手臂."
    },
    {
      "en": "The market might abandon the stock, and knock down its price.",
      "cn": "市场可能会抛售该股票，从而令其股价下跌。"
    },
    {
      "en": "The officers and crew prepared to abandon ship in an orderly fashion.",
      "cn": "全体船员秩序井然地准备弃船。"
    },
    {
      "en": "Their decision to abandon the trip was made because of financial constraints.",
      "cn": "他们决定放弃这次出游是因为财力有限。"
    },
    {
      "en": "Their permissiveness to-ward their children reflects the wild abandon of their own lives.",
      "cn": "他们对孩子的放任折射出他们自身生活的放浪形骸。"
    },
    {
      "en": "They abandon themselves to drinking.",
      "cn": "他们沉湎于饮酒."
    },
    {
      "en": "They abandoned their attempt to recapture the castle.",
      "cn": "他们放弃了夺回城堡的努力。"
    },
    {
      "en": "We had to abandon the car and walk the rest of the way.",
      "cn": "我们只好弃车，步行走完剩下的路。"
    },
    {
      "en": "We will never abandon our principles.",
      "cn": "我们应该坚持原则."
    }
  ],
  "rel_words": [
    {
      "pos": "adj",
      "words": [
        {
          "word": "abandoned",
          "tran_cn": " 被抛弃的；无约束的；恣意放荡的；寡廉鲜耻的"
        }
      ]
    },
    {
      "pos": "n",
      "words": [
        {
          "word": "abandonment",
          "tran_cn": " 抛弃；放纵"
        }
      ]
    },
    {
      "pos": "v",
      "words": [
        {
          "word": "abandoned",
          "tran_cn": " 抛弃（abandon的过去式和过去分词）"
        }
      ]
    }
  ],
  "synos": [
    {
      "pos": "n",
      "words": [
        {
          "words": [
            "loose",
            "mania"
          ],
          "tran_cn": "狂热；放任"
        }
      ]
    },
    {
      "pos": "other",
      "words": [
        {
          "words": [
            "abdicate",
            "cease",
            "cede",
            "depart",
            "desert",
            "desist",
            "discard",
            "discontinue",
            "drop",
            "evacuate",
            "forsake",
            "freedom",
            "leave",
            "quit",
            "relinquish",
            "renounce",
            "resign",
            "surrender",
            "withdraw"
          ],
          "tran_cn": "其他释义"
        }
      ]
    },
    {
      "pos": "vt",
      "words": [
        {
          "words": [
            "cease",
            "depart",
            "desert",
            "discard",
            "discontinue",
            "entirely",
            "evacuate",
            "forsake",
            "give",
            "leave",
            "quit",
            "relinquish",
            "surrender",
            "undone",
            "up",
            "withdraw"
          ],
          "tran_cn": "放弃,中止;离开"
        }
      ]
    }
  ],
  "phrases": [
    {
      "phrase": "abandon ship",
      "tran_cn": "弃船"
    },
    {
      "phrase": "with abandon",
      "tran_cn": "恣意地，放纵地"
    }
  ],
  "trans": [
    {
      "pos": "n",
      "tran_cn": "放任，放纵",
      "tran_en": "if someone does something with abandon, they behave in a careless or uncontrolled way, without thinking or caring about what they are doing"
    },
    {
      "pos": "v",
      "tran_cn": "抛弃，舍弃，中止，放弃，放纵",
      "tran_en": "to leave someone, especially someone you are responsible for"
    },
    {
      "pos": "vt",
      "tran_cn": "离弃，丢弃；遗弃，抛弃；完全放弃",
      "tran_en": "to leave someone, especially someone you are responsible for"
    }
  ],
  "exchange": {
    "third": [
      "abandons"
    ],
    "past": [
      "abandoned"
    ],
    "done": [
      "abandoned"
    ],
    "ing": [
      "abandoning"
    ]
  }
}