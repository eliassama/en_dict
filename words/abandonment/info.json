{
  "word": "abandonment",
  "ipa": {
    "us": "əˈbændənmənt",
    "uk": "əˈbændənmənt"
  },
  "sentences": [
    {
      "en": "...memories of her father's complete abandonment of her.",
      "cn": "…她父亲对她的彻底抛弃的那些记忆。"
    },
    {
      "en": "Abandonment appears in the ship insurance contract of marine insurance first.",
      "cn": "委付最初出现于海上保险中的船舶保险合同中."
    },
    {
      "en": "Cathy's experience of abandonment is hard for a Martian to understand.",
      "cn": "男人难以了解卡丝被遗弃的感觉."
    },
    {
      "en": "Constant rain forced the abandonment of the next day's competitions.",
      "cn": "连绵阴雨致使翌日各项赛事被迫中止。"
    },
    {
      "en": "However, two consecutive years of non - use raise a presumption of abandonment.",
      "cn": "但是, 连续两个未使用将导致视为放弃."
    },
    {
      "en": "In its most general sense, corruption means the perversion or abandonment.",
      "cn": "就其最一般的意义上说, 舞弊就是堕落,就是背离准则."
    },
    {
      "en": "Last week seemed to have Been spent in reckless aBandonment.",
      "cn": "上星期像是在不顾后果的放任中度过的."
    },
    {
      "en": "Organized abandonment gives them a way out of the morass.",
      "cn": "有组织的放弃给他们指明了离开这片沼泽的出路."
    },
    {
      "en": "She wept at once, with sudden, wild abandonment, in her sister's arms.",
      "cn": "她猛地一下倒在姐姐怀里“哇 ” 地一声号啕大哭起来."
    },
    {
      "en": "The causes and patterns of child abandonment in China are complicated.",
      "cn": "儿童遗弃的类型和原因非常复杂."
    },
    {
      "en": "The death penalty was abolished; the abandonment of nuclear weapons confirmed.",
      "cn": "甚至, 环保在那时也得到了发展."
    },
    {
      "en": "The grounds for abandonment may include injury ascribable to the opponent's actions.",
      "cn": "有可能因伤而放弃比赛并非由于对手的行为."
    },
    {
      "en": "The way of life is wonderful: it is by abandonment.",
      "cn": "生命之路是精彩的,而这种精彩来自放弃."
    },
    {
      "en": "There is a downside to the higher sales, however: increased abandonment.",
      "cn": "不过, 宠物狗市场行情看俏也有它的缺点: 弃养的情况增加了."
    },
    {
      "en": "You will also face heartbreak and abandonment by those you love.",
      "cn": "你同样会面临心碎时刻以及被深爱的人抛弃."
    },
    {
      "en": "You're caught up in three devastating feelings: hurt, abandonment, and doubt.",
      "cn": "你的感受有三部分 —— 受伤害 、 被忽视和怀疑上帝."
    }
  ],
  "rel_words": [
    {
      "pos": "adj",
      "words": [
        {
          "word": "abandoned",
          "tran_cn": " 被抛弃的；无约束的；恣意放荡的；寡廉鲜耻的"
        }
      ]
    },
    {
      "pos": "n",
      "words": [
        {
          "word": "abandon",
          "tran_cn": " 狂热；放任"
        }
      ]
    },
    {
      "pos": "v",
      "words": [
        {
          "word": "abandoned",
          "tran_cn": " 抛弃（abandon的过去式和过去分词）"
        }
      ]
    },
    {
      "pos": "vt",
      "words": [
        {
          "word": "abandon",
          "tran_cn": " 遗弃；放弃"
        }
      ]
    }
  ],
  "synos": [
    {
      "pos": "n",
      "words": [
        {
          "words": [
            "rejection",
            "loose"
          ],
          "tran_cn": "[法]抛弃；放纵"
        }
      ]
    },
    {
      "pos": "other",
      "words": [
        {
          "words": [
            "cession",
            "desertion",
            "exposition",
            "exposure",
            "rejection",
            "resignation",
            "retirement"
          ],
          "tran_cn": ""
        }
      ]
    }
  ],
  "phrases": [],
  "trans": [
    {
      "pos": "n",
      "tran_cn": "放弃，抛弃，遗弃，放任",
      "tran_en": "The abandonment of a place, thing, or person is the act of leaving it permanently or for a long time, especially when you should not do so"
    }
  ],
  "exchange": {}
}