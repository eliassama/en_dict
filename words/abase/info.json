{
  "word": "abase",
  "ipa": {
    "us": "əˈbeɪs",
    "uk": "əˈbes"
  },
  "sentences": [
    {
      "en": "A man who never respect others will abase hisself.",
      "cn": "不尊重别人的人同时也自贬了身份."
    },
    {
      "en": "A man who uses bad language will only abase himself.",
      "cn": "说脏话者只会自贬身分."
    },
    {
      "en": "Abase all, the Server application and Client application implementation was followed.",
      "cn": "在上述的基础上,编写了实现应用服务器和客户端应用程序的方法、步骤."
    },
    {
      "en": "Don't abase your apartment.",
      "cn": "不要降低你的地位。"
    },
    {
      "en": "Don't abase yourself! You're unique, you can do everything!",
      "cn": "不许贬低自己! 你是独一无二的! 你可以做任何事!"
    },
    {
      "en": "Finally, the open problems on research in Dat abase Security are discussed.",
      "cn": "最后提出了数据库安全研究应当解决的几个问题."
    },
    {
      "en": "He refused to abase himself in the eyes of others.",
      "cn": "他不愿在他人面前被贬低."
    },
    {
      "en": "He would not abase himself by showing fear.",
      "cn": "他不愿露出惧色而有失身份."
    },
    {
      "en": "In former times, priests used to abase themselves before the gods.",
      "cn": "过去, 牧师们常常对神自责."
    },
    {
      "en": "She wanted to abase herself before the strong man, to be low and humble before him.",
      "cn": "她使自己在这个强壮的男人面前显得更谦卑,低下和恭顺."
    },
    {
      "en": "The leader's refusal to abase himself before his followers irritated the king.",
      "cn": "这位首领拒绝在他的部下面前低三下四,这使得皇帝很恼火."
    }
  ],
  "rel_words": [
    {
      "pos": "n",
      "words": [
        {
          "word": "abasement",
          "tran_cn": " 身分低微；屈尊；降低"
        }
      ]
    }
  ],
  "synos": [
    {
      "pos": "other",
      "words": [
        {
          "words": [
            "bring",
            "degrade",
            "demote",
            "dishonor",
            "down",
            "impute",
            "reduce",
            "shame"
          ],
          "tran_cn": "其他释义"
        }
      ]
    },
    {
      "pos": "vt",
      "words": [
        {
          "words": [
            "bring",
            "debase",
            "degrade",
            "demean",
            "demote",
            "down",
            "humble",
            "humiliate",
            "lower",
            "make",
            "reduce"
          ],
          "tran_cn": "贬低,降低;使受辱"
        }
      ]
    }
  ],
  "phrases": [],
  "trans": [
    {
      "pos": "v",
      "tran_cn": "降低",
      "tran_en": "to humble or belittle (oneself, etc) "
    },
    {
      "pos": "vt",
      "tran_cn": "使谦卑，使感到羞耻，使降低（地位、身份等），<古>降下"
    }
  ],
  "exchange": {
    "third": [
      "abases"
    ],
    "past": [
      "abased"
    ],
    "done": [
      "abased"
    ],
    "ing": [
      "abasing"
    ],
    "noun": [
      "abasement"
    ]
  }
}