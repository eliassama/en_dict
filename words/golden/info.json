{
  "word": "golden",
  "ipa": {
    "us": "ˈɡəʊldən",
    "uk": "ˈɡoʊldən"
  },
  "sentences": [
    {
      "en": "Bake in the preheated oven for 25 minutes or until golden brown.",
      "cn": "在预热过的烤箱里烤25分钟，或烤成金黄色。"
    },
    {
      "en": "Drop the slices into the oil and fry until golden brown.",
      "cn": "将切片倒进油里，煎至金棕色。"
    },
    {
      "en": "golden hair",
      "cn": "金发"
    },
    {
      "en": "golden sand",
      "cn": "金色的沙"
    },
    {
      "en": "Hanson's golden rule is to add value to whatever business he buys.",
      "cn": "汉森的成功原则是不论收购什么样的企业都要使其增值。"
    },
    {
      "en": "He stood there in a cocoon of golden light.",
      "cn": "他站在那儿，一柱金光笼罩着他。"
    },
    {
      "en": "He was gently caressing her golden hair.",
      "cn": "他深情抚摸着她的金发。"
    },
    {
      "en": "He wasted a golden opportunity when he missed from the penalty spot.",
      "cn": "他罚球没有命中，浪费了一个绝佳的机会。"
    },
    {
      "en": "If you remember these three golden rules you won't go far wrong.",
      "cn": "记住这三条金律，就不会错到哪儿。"
    },
    {
      "en": "If you're lucky, a splendid golden eagle may soar into view.",
      "cn": "幸运的话，会有一只光彩夺目的金雕飞入你的视野。"
    },
    {
      "en": "Imagine long golden beaches where you can wander in solitude.",
      "cn": "想象一下那长长的金色海滩吧，在那里你可以独自徜徉。"
    },
    {
      "en": "In the light from the hall, her hair had a golden gleam.",
      "cn": "在大厅灯光的照射下，她的头发闪着金色的光芒。"
    },
    {
      "en": "Not a strand of her golden hair was out of place.",
      "cn": "她的金发纹丝不乱。"
    },
    {
      "en": "Orange and khaki flatter those with golden skin tones.",
      "cn": "橘黄色和卡其色能衬托那些金黄色皮肤的人。"
    },
    {
      "en": "She chopped off her golden, waist-length hair.",
      "cn": "她剪掉了她那齐腰的金发。"
    },
    {
      "en": "She combed and arranged her golden hair.",
      "cn": "她梳理好自己的金发。"
    },
    {
      "en": "The golden bangles on her arms clashed and jingled.",
      "cn": "她手臂上的金手镯丁零当啷地响。"
    },
    {
      "en": "The golden rule is never to clean a valuable coin.",
      "cn": "重要的原则是决不清除贵重硬币上的污垢。"
    }
  ],
  "rel_words": [
    {
      "pos": "adj",
      "words": [
        {
          "word": "gold",
          "tran_cn": " 金的，金制的；金色的"
        }
      ]
    },
    {
      "pos": "n",
      "words": [
        {
          "word": "gold",
          "tran_cn": " 金，黄金；金色；金币"
        }
      ]
    }
  ],
  "synos": [
    {
      "pos": "adj",
      "words": [
        {
          "words": [
            "metallic"
          ],
          "tran_cn": "金的;金属的"
        },
        {
          "words": [
            "bright",
            "shining"
          ],
          "tran_cn": "金色的;光亮的"
        },
        {
          "words": [
            "excellent",
            "good",
            "splendid",
            "superior",
            "valuable",
            "very"
          ],
          "tran_cn": "高贵的;优秀的"
        }
      ]
    },
    {
      "pos": "other",
      "words": [
        {
          "words": [
            "bright",
            "excellent",
            "fine",
            "good",
            "metallic",
            "nice",
            "shining",
            "splendid",
            "superior",
            "valuable",
            "very",
            "yellow"
          ],
          "tran_cn": "其他释义"
        }
      ]
    }
  ],
  "phrases": [
    {
      "phrase": "golden age",
      "tran_cn": "黄金时代；鼎盛时期"
    },
    {
      "phrase": "golden ball",
      "tran_cn": "金球；金琥"
    },
    {
      "phrase": "golden bell",
      "tran_cn": "连翘"
    },
    {
      "phrase": "golden boy",
      "tran_cn": "有成就的男人"
    },
    {
      "phrase": "golden bridge",
      "tran_cn": "退路（逃路）"
    },
    {
      "phrase": "golden brown",
      "tran_cn": "金黄色；淡棕色"
    },
    {
      "phrase": "golden eagle",
      "tran_cn": "金雕"
    },
    {
      "phrase": "golden gate",
      "tran_cn": "金门海峡（美国一海峡）"
    },
    {
      "phrase": "golden gate bridge",
      "tran_cn": "金门大桥"
    },
    {
      "phrase": "golden hair",
      "tran_cn": "金发"
    },
    {
      "phrase": "golden key",
      "tran_cn": "贿赂"
    },
    {
      "phrase": "golden mean",
      "tran_cn": "中庸之道；[美]黄金分割（等于golden section）"
    },
    {
      "phrase": "golden monkey",
      "tran_cn": "金丝猴"
    },
    {
      "phrase": "golden opportunity",
      "tran_cn": "绝好的机会"
    },
    {
      "phrase": "golden rule",
      "tran_cn": "金科玉律；指导原则；黄金法则"
    },
    {
      "phrase": "golden section",
      "tran_cn": "[数]黄金分割"
    },
    {
      "phrase": "golden state",
      "tran_cn": "（美）加利福尼亚州的别名"
    },
    {
      "phrase": "golden triangle",
      "tran_cn": "金三角"
    },
    {
      "phrase": "golden yellow",
      "tran_cn": "金黄；金黄色"
    },
    {
      "phrase": "silence is golden",
      "tran_cn": "沈默是金"
    }
  ],
  "trans": [
    {
      "pos": "adj",
      "tran_cn": "金（黄）色的，金质的，美好的 ",
      "tran_en": "having a bright yellow colour like gold"
    },
    {
      "pos": "n",
      "tran_cn": "(Golden)人名；(英、法、罗、德、瑞典)戈尔登"
    }
  ],
  "exchange": {}
}